src/aks-preview/azext_aks_preview/decorator.py

```diff
@@ -1853,6 +1867,26 @@ def get_workload_identity_profile(self) -> Optional[ManagedClusterSecurityProfil
 
         return profile
 
+    def get_web_app_routing_profile(self) -> Optional[ManagedClusterIngressProfileWebAppRouting]:
+        """Obtrain the value of ingress_profile.web_app_routing.
+
+        :return: Optional[ManagedClusterIngressProfileWebAppRouting]
+        """
+        dns_zone_resource_id = self.raw_param.get("dns_zone_resource_id")
+
+        profile = self.models.ManagedClusterIngressProfileWebAppRouting()
+        profile.enabled = True
```

Comment:
It seems that the web app routing addon is enabled by default.

May refer to the [checks](https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/azext_aks_preview/decorator.py#L2387-L2391) performed by gitops addon. I suppose something like
```
if "web_application_routing" in self.context.get_enable_addons():
    xxxx
```

Reply:
Fixed